demisto_sdk/commands/common/tools.py

```diff
@@ -3385,3 +3408,31 @@ def field_to_cli_name(field_name: str) -> str:
         field_name (str): the incident/indicator field name.
     """
     return re.sub(NON_LETTERS_OR_NUMBERS_PATTERN, "", field_name).lower()
+
+
+def get_path_in_content_repo(file_path):
```

Comment:
please use pathlib instead

Reply:
p.s. don't we have such a function already?

Reply:
We have `get_relative_path_from_packs_dir()` which is not using pathlib as well

Reply:
please make it use pathlib and use it, you can pretty much copy the code from here (note the exception class is customized there, so we need to either return _something_ (`None`?) or raise a separate exception - same as `get_relative_path_from_packs_dir`